_drafts/main.md

```diff
@@ -0,0 +1,205 @@
+
+Anyway, here's a Python module I made.
+
+### Introducing: `factorio-draftsman`
+
+{% include image.html src='https://media.alt-f4.blog/ALTF4/61/draftsman_logo.jpg' alt='Draftsman logo, represented in-game, generated via script.' caption='I went through dozens of different combinations of tile orders before settling on this one.' %}
+
+[Draftsman](https://github.com/redruin1/factorio-draftsman) is a Python module for creating, modifying, importing, and exporting all manner of Factorio blueprint strings. The package allows you to create and design blueprints programmatically, to aid in the development of tedious and repetitive blueprints that would be untenable to create by hand, much like the problem I ran into described above. Draftsman attempts to solve all of the shortcomings of existing Factorio blueprint implementations:
+
+* **Draftsman does it all.** All types of entities are supported, from splitters to stack inserters. If you can do it in-game, you can do it with Draftsman, allowing you to focus on the one problem you're *actually* trying to solve.
+* **Draftsman is uni-language.** Written in Python, this makes it exceptionally easy to install, simple to use, and gives the user access to the entirety of Python's vast packaging index. Chances are you can do it in Python with Draftsman, regardless of *what* Factorio-related thing you're actually doing.
+* **Draftsman is easy to use.** Designed from the start to be simple and (most importantly) self-documenting, Draftsman allows you to manipulate blueprints and entities by auto-completable attributes and methods.
+* **Draftsman is well-documented.** Every function, method, attribute, and class is documented and crosslinked at its [readthedocs site](https://factorio-draftsman.readthedocs.io/en/latest/?badge=latest). In addition, tutorials and supplementary materials are provided, as well as a whole host of [example programs](https://github.com/redruin1/factorio-draftsman/tree/main/examples) to help illustrate how Draftsman works.
+* **Draftsman is stable.** A rigorous suite of tests ensure that Draftsman behaves predictably and correctly (or at least correctly enough), with a code coverage of 100%. Draftsman is verified to work on the latest versions of Python 2 and 3, and is compatible with all versions of Factorio greater than 1.0, with the metrics to prove it.
+* **Draftsman is descriptive.** Draftsman enforces "Factorio-safety" as a core philosophy, which means that if a blueprint would result in an error on import in Factorio, an exception is thrown by Draftsman. Draftsman also tries to enforce "Factorio-correctness", which means that values that won't break, but are otherwise nonsensical, will raise warnings. Both errors and warnings are verbose, so that any problem with your script can be understood and resolved in seconds.
```

Comment:
```suggestion
* **Draftsman is descriptive.** Draftsman enforces "Factorio-safety" as a core philosophy, which means that if a blueprint would result in an import error in Factorio, an exception is thrown by Draftsman. Draftsman also tries to enforce "Factorio-correctness", which means that values that won't break, but are otherwise nonsensical, will raise warnings. Both errors and warnings are verbose, so that any problem with your script can be understood and resolved in seconds.
```